cartography/intel/aws/iam.py

```diff
@@ -248,9 +248,32 @@ def get_role_list_data(boto3_session: boto3.session.Session) -> Dict:
     roles: List[Dict] = []
     for page in paginator.paginate():
         roles.extend(page['Roles'])
+    for role in roles:
+        role_data = client.get_role(RoleName=role.get("RoleName")).get("Role")
+        role['RoleLastUsed'] = role_data.get("RoleLastUsed")
     return {'Roles': roles}
 
 
+@timeit
+def get_external_access_roles(boto3_session: boto3.session.Session) -> List[Dict]:
+    try:
+        analyzer_client = boto3_session.client('accessanalyzer')
+        # AWS allow to create only one analyzer per account per region with type (ACCOUNT, ORGANIZATION, ACCOUNT_UNUSED_ACCESS, and ORGANIZTAION_UNUSED_ACCESS) thats why uses analyzers_list index 0
+        analyzers_list = analyzer_client.list_analyzers(type='ACCOUNT').get("analyzers", [])
+        findings: List = []
+        if analyzers_list:
+            paginator = analyzer_client.get_paginator('list_findings_v2')
+            for page in paginator.paginate(analyzerArn=analyzers_list[0].get("arn"), filter={'resourceType': {'eq': ['AWS::IAM::Role']}}):
+                findings.extend(page.get("findings", []))
+            return findings
+        else:
+            analyzer_client.create_analyzer(analyzerName="cdx_analyzer", type="ACCOUNT", tags={'owner': 'cdx_lab', 'project': 'iam-entitlement'})
```

Comment:
let's change the tag values.
owner should be cloudanix
project - iam-entitlements-analyzer